test/integration/endpoints/endpoints_test.go

```diff
@@ -156,6 +156,160 @@ func TestEndpointUpdates(t *testing.T) {
 
 }
 
+// TestEndpointWithTerminatingPod tests that terminating pods are NOT included in Endpoints
+// DO NOT update this test to include terminating endpoints. This test ensures that Endpoints
```

Comment:
`DO NOT update this test to include terminating endpoints` is a bit confusing to me, maybe just remove this or add a comment that's more similar to your PR description.

Reply:
That's fair, updated!